src/ubuntu/stage-upgrade-mainnet.md

```diff
@@ -1,105 +1,4 @@
 ### Stage the upgrades
```

Comment:
This seems redundant now. I think either this section or the previous subtitle `Stage all protocol upgrades` needs to be removed.

Reply:
Removed staging-upgrade-mainnet.md from the repo and removed the include statement in the base readme.